UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs

```diff
@@ -98,16 +98,10 @@ private IEnumerator Coitus(PlayerHealthV2 player)
 		private async Task Pregnancy(PlayerHealthV2 player)
 		{
 			KillHugger();
-			await Task.Delay(TimeSpan.FromSeconds(pregnancyTime));
-			//TODO check if the larvae was removed from stomach
-			player.ApplyDamageToBodyPart(
-				gameObject,
-				200,
-				AttackType.Internal,
-				DamageType.Brute,
-				BodyPartType.Chest);
-
-			Spawn.ServerPrefab(larvae, player.gameObject.RegisterTile().WorldPositionServer);
+
+			GameObject embryo = Spawn.ServerPrefab(larvae, SpawnDestination.At(gameObject), 1).GameObject;
+
+			player.BodyPartList[6].OrganStorage.ServerTryAdd(embryo); //6 is torso
```

Comment:
This assumes that the correct part is always index 6. There might be a better way to get the correct boy part, @Bod9001 will know.

Reply:
I also want to get the boy part uwu

Reply:
There was a chance that no one would see that typo

Reply:
> NEVER USE HARDCODED [insert type here] FOR SEARCHING FOR ANYTHING BECAUSE THEY WILL CHANGE SOMETIME IN THE FUTURE WITHER YOU LIKE THAT OR NOT

This was me like a month or two ago, don't make me yell again in a PR about hardcoded stuff.

Reply:
best way to do it would be Loop through, the entire organist and look for the one with the tag torso?

Reply:
could also just put it in the stomach, though you would need to add the surgery for opening stomach

Reply:
I thought it was intended to go inside the stomach

Reply:
currently it's just going inside of the torso

Reply:
Ill chuck it in the stomach, will just have to add those surgery steps

Reply:
> There have been multiple theories regarding the 'impregnation process' and what it is that the Facehugger implants into its host. The first and initially the most common theory was that once subdued, the host's oesophagus is invaded with the parasite's proboscis, which allows the facehugger to implant an embryo. This is also accompanied by a chemical mutagen which may allow for a 'placenta' to form in the host.[[citation needed](http://en.wikipedia.org/wiki/Citation_needed)] The main contention with the embryo theory was that since Chestbursters gestate within the chest cavity of the host, there would be no access to that area from within the stomach or oesophagus, which would be where an embryo would be deposited.

🤔

Reply:
Hmmmm so what they saying is it gets deposited in the stomach then space magic makes it go to the torso... I'll chuck it in the stomach anyways, it makes more sense and we actually have a function GetStomachs() which will stop this being hardcoded